concent-builder/configure.yml

```diff
@@ -50,25 +55,26 @@
           with_items:
             - build-essential
             - python2.7
-            - python3
-            - python-pip
+            - python3.6
+            - python3.6-dev
+            - python3.6-venv
             - libssl-dev
+            - pkg-config
+            - libsecp256k1-dev
             - git
             - kubectl
             - docker-engine
             - google-cloud-sdk
             - ansible
-            - virtualenv
             - libpq-dev
             - gcc
-            - python3-dev
             - python3-yaml
 
         - name:     apt autoclean
           command:  apt autoclean
 
         - name:     apt autoremove
-          command:  apt autoremove
+          command:  apt --assume-yes autoremove
```

Comment:
Always put the extra options after the command. This makes it easier to see that the command is:
``` bash
apt autoremove --assume-yes
```